### PR TITLE
License, copyright, and README cleanup in all templates

### PR DESCRIPTION
**Pull request by kwalrath**

### CODE REVIEW

File: templates/console-full/README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # __projectName__
 
 A sample command-line application.
+
+Generated by Stagehand.
+See the [LICENSE](https://github.com/dart-lang/stagehand/blob/master/LICENSE).
```
Comment (agamdua) [inline templates/console-full/README.md]: As a user and external contributor who will add the obligatory "IANAL".. The language is ambiguous whether this is the license for _stagehand_ or the generated project.

Is there - possibly - a way to make this less ambiguous?

Comment (kwalrath) [reply templates/console-full/README.md]: This wording was approved, but that's a good point, I'll check on that.

Comment (agamdua) [reply templates/console-full/README.md]: Thanks @kwalrath for the quick response!

A project generator should in no way **assign the rights of _my_ project away** to any other entity. I really hope that the legal team agrees here.

To emphasize the (still relevant) concerns in https://github.com/dart-lang/stagehand/pull/450#issuecomment-335889618, such a change will make this project absolutely unusable to anyone really.

Comment (kwalrath) [reply templates/console-full/README.md]: We've changed the wording to clarify that it's the template code that's BSD licensed.

**Pull request merged**